chain-abci/fuzz/Cargo.lock

```diff
@@ -0,0 +1,3908 @@
+# This file is automatically @generated by Cargo.
```

Comment:
needed? or could this go to .gitignore?

Reply:
I think general guideline is like libraries should ignore Cargo.lock but binaries/applications should check-in Cargo.lock, I guess fuzz can be considered as application?

Reply:
Maybe we can put the fuzz crate in workspace, it'll share the same Cargo.lock file?

Reply:
Removed the lock file.